src/python/pants/notes/master.rst

```diff
@@ -4,6 +4,77 @@ Master Pre-Releases
 This document describes development releases which occur weekly from master, and which have
 not undergone the vetting associated with ``stable`` releases.
 
+1.17.0.dev0 (5/13/2019)
+-----------------------
+
+New Features
+~~~~~~~~~~~~
+
+* V2 `./pants test.pytest` selects interpreter based off of compatibility constraints (#7679)
+  `PR #7679 <https://github.com/pantsbuild/pants/pull/7679>`_
+
+Bugfixes
+~~~~~~~~
+
+* Mark build-support Python files as Pants targets to lint build-support (#7633)
+  `PR #7633 <https://github.com/pantsbuild/pants/pull/7633>`_
+
+* Fix dependency cycle with cache folder (#7693)
+  `PR #7693 <https://github.com/pantsbuild/pants/pull/7693>`_
+
+* Add pantsd_invalidation_globs option into pants.ini (#7705)
```

Comment:
While it's a bug fix, it only impacts our own `pants.ini` so I would suggest moving it to tooling.